pyanaconda/payload/dnf/payload.py

```diff
@@ -2045,6 +2046,18 @@ def post_install(self):
             except PayloadSetupError as e:
                 log.error(e)
 
+        # rpm needs importing installed certificates manually, see rhbz#748320 and rhbz#185800
+        if conf.payload.import_keys_to_rpm:
+            log.info("Importing keys to RPM database.")
+            #releasever = util.execWithCapture("rpm", ["--eval", "'%{fedora}'"])
+            #basearch = util.execWithCapture("uname", ["-i"])
+            util.execInSysroot("rpm", [
+                "--import", "/etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-{releasever}-{basearch}".format(
+                    releasever=util.execWithCapture("rpm", ["--eval", "'%{fedora}'"]),
+                    basearch=get_arch()
```

Comment:
I think you can use `_replace_vars` for the substitution.

Reply:
Then the Fedora configuration file could set the option for the gpg keys to:

`default_gpg_keys = /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch`

Reply:
Done. See below / diff for the aftermath.